jakarta/quarkus3.yml

```diff
@@ -0,0 +1,984 @@
+#
+# Copyright 2021 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.Quarkus3
+displayName: Migrate project to Quarkus 3
+description: Update Quarkus version
+tags:
+  - quarkus
+recipeList:
+  - org.openrewrite.java.migrate.JavaxMigrationToJakarta
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: quarkus.platform.version
+      newValue: 3.0.0.Alpha1
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: quarkus.version
+      newValue: 3.0.0.Alpha1
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JavaxMigrationToJakarta
```

Comment:
You have to copy the recipe here? You cannot refer to the one bundled in OpenRewrite (I suppose it's bundled)? Just asking, I have no idea if it works.

Reply:
can you give it a try, it works locally, but maybe I have it somewhere already..

Reply:
@gsmet sorry I misunderstood you, no you can't refer to the other one AFAIK. Also, I've add to comment a few stuff inside it as it doesn't apply to Quarkus (Add dependencies)